lib/get-repositories.js

```diff
@@ -1,9 +1,24 @@
 const fs = require('fs');
+const { isArray } = require('lodash');
 const { GITHUB_REPO_REGEX, extractOwnerAndRepo } = require('./github-helpers');
 
-const getRepositories = limit => {
-	const input = fs.readFileSync('/dev/stdin') || '';
-	const inputRepositories = input.toString().split('\n');
+const getRepositories = ({ limit, repoList } = {}) => {
+	let inputRepositories;
+
+	const hasStdIn = !process.stdin.isTTY;
+
+	if (isArray(repoList) && !!repoList.length && hasStdIn) {
```

Comment:
We can be more clear about this condition by not using `!!`.

```suggestion
	if (isArray(repoList) && repoList.length === 0 && hasStdIn) {
```

Reply:
I've used Array.isArray 👍

Reply:
Ahaha, that was totally not clear to me 😅